Bottom: c85a208244efaa12226003dbaf0478df35810d30
Top:    0cc50d4fd1a993eac5c5909ae4cb8b39a54a91e7
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-02 21:40:12 +0100

Adjust punctuation and capitalisation of some messages


---

diff --git a/src/hid/gtk/gui-keyref-window.c b/src/hid/gtk/gui-keyref-window.c
index 5037bb0..801d6d7 100644
--- a/src/hid/gtk/gui-keyref-window.c
+++ b/src/hid/gtk/gui-keyref-window.c
@@ -296,7 +296,7 @@ static gchar *key_ref_text[] = {
   N_(" - pan vertically.\n"),
   "\t",
   N_("<b><ctrl>"),
-  N_(" - pan horizontally..\n"),
+  N_(" - pan horizontally.\n"),
   "\n",
   N_("<b>Usage:\n"),
   N_("\tMouse actions can typically be combined.  For example: while moving\n"
diff --git a/src/rats.c b/src/rats.c
index c174cc9..0e2a068 100644
--- a/src/rats.c
+++ b/src/rats.c
@@ -397,7 +397,7 @@ CheckShorts (LibraryMenuTypePtr theNet)
 	  {
 	    menu = GetPointerMemory (generic);
 	    *menu = pin->Spare;
-	    Message (_("Warning! net \"%s\" is shorted to net \"%s\"\n"),
+	    Message (_("Warning! Net \"%s\" is shorted to net \"%s\"\n"),
 		     &theNet->Name[2],
 		     &((LibraryMenuTypePtr) (pin->Spare))->Name[2]);
 	    SET_FLAG (WARNFLAG, pin);
